workflow/scripts/neighborhoods.py

```diff
@@ -457,21 +458,15 @@ def count_bam(
 
 def count_tagalign(tagalign, bed_file, output, genome_sizes):
     index_file = tagalign + ".tbi"
-    if os.path.exists(index_file):
-        command1 = ""
-    else:
-        command1 = "tabix -p bed {tagalign} | cut -f1-3".format(**locals())
-
-    command2 = 'bedtools coverage -counts -b {tagalign} -a {bed_file} | awk \'{{print $1 "\\t" $2 "\\t" $3 "\\t" $NF}}\' '.format(
-        **locals()
-    )
+    if not os.path.exists(index_file):
+        cmd = f"tabix -p bed {tagalign} | cut -f1-3"
+        check_call(cmd, shell=True)
 
-    p1 = Popen(command1, stdout=PIPE, shell=True)
+    coverage_cmd = f"bedtools coverage -counts -b {tagalign} -a {bed_file}"
```

Comment:
ended up removing `sorted` flag for now because it doesn't work out of the box. The tagAlign file has lots of alternative chromosome which aren't in the chromosomes sizes file, which causes issues. To use sorted, we would have to filter out the original tagAlign's alternative chromosome